dockstore-webservice/src/main/java/io/dockstore/webservice/core/Collection.java

```diff
@@ -55,7 +55,7 @@
     @Schema(description = "Implementation specific ID for the collection in this web service")
     private long id;
 
-    @Column(nullable = false, unique = true)
```

Comment:
does the unique = true just not do anything?

Reply:
It would, but it would be redundant.
i.e. setting a `unique lower(name)`  index is more stringent than a `unique name` index.
But the former can't be done via JPA